test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNetCore/AspNetCoreWebApi21Tests.cs

```diff
@@ -0,0 +1,26 @@
+using Datadog.Trace.ClrProfiler.IntegrationTests.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AspNetCore
+{
+    public class AspNetCoreWebApi21Tests : AspNetCoreMvcTestBase
+    {
+        public AspNetCoreWebApi21Tests(ITestOutputHelper output)
+            : base("AspNetCoreWebApi21", output)
+        {
+            Expectations.Clear();
```

Comment:
Not the most clean, but the `AspNetCoreMvcTestBase` was built for the common `AspNetCoreMvcXXX` samples that have a predefined set of routes, which these new samples do not have. Rather than support them, I wanted to just support a minimal set of new routes created by the `dotnet new` templates.